.travis.yml

```diff
@@ -3,8 +3,8 @@ language: c
 matrix:
   include:
     - os: linux
-    - os: osx
-      osx_image: xcode7.3
+#   - os: osx
```

Comment:
The better way to go is to allow the failures but still build so that we know about them.
https://improveandrepeat.com/2014/01/travis-ci-and-allowed-failures/